tigerpath/majors_and_certificates/majors/MUS.json

```diff
@@ -153,7 +153,17 @@
             "MUS 4*",
             "MUS 5*"
           ]
+        },
+        {
+          "name": "Music Performance",

```

Comment:
https://music.princeton.edu/sites/default/files/ua_department_of_music_2020.pdf

Music majors in the Program in Music Performance certificate may use one performance course (such as MPP 213, 214, 215, 216, 219) as a departmental.